dist/component-ear/pom.xml

```diff
@@ -0,0 +1,90 @@
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.hawkular.metrics</groupId>
+    <artifactId>hawkular-metrics-dist</artifactId>
+    <version>0.20.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>hawkular-metrics-component-dist</artifactId>
+  <packaging>ear</packaging>
+
+  <name>Hawkular Metrics Distribution for Hawkular Services</name>
+  <description>Hawkular Metrics Distribution (EAR) for Hawkular Services</description>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.hawkular.metrics</groupId>
+      <artifactId>hawkular-metrics-component</artifactId>
+      <version>${project.version}</version>
+      <type>war</type>
+    </dependency>
+    <dependency>
+      <groupId>org.hawkular.alerts</groupId>
+      <artifactId>hawkular-alerts-rest</artifactId>
+      <version>${version.org.hawkular.alerts}</version>
+      <type>war</type>
+    </dependency>
+    <dependency>
+      <groupId>org.hawkular.alerts</groupId>
+      <artifactId>hawkular-alerter-metrics</artifactId>
+      <version>${version.org.hawkular.alerts}</version>
+      <type>war</type>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-ear-plugin</artifactId>
+        <configuration>
+          <version>7</version>
+          <initializeInOrder>true</initializeInOrder>
+          <fileNameMapping>no-version</fileNameMapping>
+          <modules>
+            <webModule>
+              <groupId>org.hawkular.metrics</groupId>
+              <artifactId>hawkular-metrics-component</artifactId>
+              <bundleFileName>hawkular-metrics.war</bundleFileName>
+              <contextRoot>/hawkular/metrics</contextRoot>
+            </webModule>
+            <webModule>
+              <groupId>org.hawkular.alerts</groupId>
+              <artifactId>hawkular-alerts-rest</artifactId>
+              <bundleFileName>hawkular-alerts.war</bundleFileName>
+              <contextRoot>/hawkular/alerts</contextRoot>
+            </webModule>
+            <webModule>
+              <groupId>org.hawkular.alerts</groupId>
+              <artifactId>hawkular-alerter-metrics</artifactId>
+              <contextRoot>/hawkular/__alerter-metrics</contextRoot>
```

Comment:
I am not sure if I like this url format.
I understand that alerter is headless and having a context root is mandatory but not used.
And I guess that pattern is to warn that this is not used, but I guess we can avoid the "__" chars.